### PR TITLE
Rename image from jetty9 to jetty

### LINKED ISSUE #53: Rename image from jetty9 to jetty

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes #53

### CODE REVIEW

**Comment by meltsufin** [comment]
@aslo PTAL

File: README.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ This repository contains the source for the `gcr.io/google_appengine/jetty` [doc
 The layout of this image is intended to mostly mimic the official [docker-jetty](https://github.com/appropriate/docker-jetty) image and unless otherwise noted, the official [docker-jetty documentation](https://github.com/docker-library/docs/tree/master/jetty) should apply.
 
 ## Building the Jetty image
-To build the image you need git, docker and maven installed and to have the openjdk8:8-jre
+To build the image you need git, docker and maven installed and to have the openjdk:8
 image available in your docker repository:
 ```console
 git clone https://github.com/GoogleCloudPlatform/jetty-runtime.git
```
Comment (aslo) [inline README.md]: Should also `cd jetty-runtime` after cloning.

**Comment by aslo** [comment]
LGTM, other than one small nit

File: README.md (hunk 1 above)
Comment (meltsufin) [inline README.md]: Done

**Pull request merged**